docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.9"
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
```

Comment:
If we're eventually thinking of a Dockerfile for the frontend as well, could we differentiate these with directories, maybe something like `docker/web/Dockerfile` and `docker/frontend/Dockerfile`?

Reply:
ah right, ok, we can add sufffixes if/when we add other files